### ISSUE TITLE
Glob support

### ISSUE THREAD

**Issue by onsails** [nienbo/cache-buildkite-plugin#13]
Is it possible to define cache dirs based on glob pattern?
I am trying to do it like this:
```
    plugins:
      - gencer/cache#v2.3.0:
          backend: tarball
          key: "web-deps-cache-{{ checksum 'yarn.lock' }}"
          tarball:
            path: '/tmp/buildkite-cache'
            max: 7 # Optional. Removes tarballs older than 7 days.
          paths: [ "node_modules/", "packages/*/node_modules/" ]
```

But apparently it's being interpreted literally:
```
🔍 Locating cache: web-deps-cache-cb1335083a124ebb7ca0a007ccfb41b95a113c1a

🗑️ Deleting backups older than 7 day(s)...

tar: packages/*/node_modules: Cannot stat: No such file or directory

tar: Exiting with failure status due to previous errors

[31m🚨 Error: The plugin cache post-command hook exited with status 2[0m
```

**Comment by gencer** [comment]
Actually it is possible to use glob pattern. However, the error you see is due to hardcoded quotes we use in tarball cache function.

This will work:

```bash
tar -cf test.tar ./**/node_modules # or ./*/node_modules
```

This will **not**:

```bash
tar -cf test.tar './**/node_modules'
```

Attention to the single quotes at start and end on path.

I will work on fix.

**Comment by gencer** [comment]
@onsails  Could you please test against issue branch?:

```yaml
    plugins:
      - gencer/cache#gencer/issue13:
      ...
```

**Comment by onsails** [comment]
Yes, I can confirm, gencer/cache#gencer/issue13 handles globs correctly, thanks!

**Comment by gencer** [comment]
Glad it did worked!

Released in `v2.3.1` 🎉 You may now use `v2.3.1` instead of issue branch which is already deleted..
